src/main/java/tahub/contacts/model/ModelManager.java

```diff
@@ -75,6 +79,37 @@ public void setAddressBookFilePath(Path addressBookFilePath) {
         userPrefs.setAddressBookFilePath(addressBookFilePath);
     }
 
```

Comment:
Maybe add a `//=========== Course ==================================================================================` similar to the rest of the file

Reply:
fixed